### PR TITLE
Create api.arman.json

### PR DESCRIPTION
**Pull request by ItzArman09**

### CODE REVIEW

**Review by hackermondev — APPROVED** [review]

**Review by hackermondev — DISMISSED** [review]
CNAME record in ``foxie.arman`` cannot have slash.

**Comment by ItzArman09** [comment]
but thats my cname, got it from gh pages... What do i do. Now

File: domains/api.arman.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Arman's Api",
+  "owner": {
+    "username": "arman",
+    "email": "contact@mail.arman.is-a.dev",
+    "github": "ItzArman09"
```
Comment (MaskDuck) [inline domains/api.arman.json:6]:
```suggestion
    "username": "ItzArman09",
    "email": "contact@mail.arman.is-a.dev"
```

File: domains/arman.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "repo": "https://github.com/ItzArman09/website",
   "owner": {
     "username": "ItzArman09",
-    "email": "gyfufifa@norwegischlernen.info"
+    "email": "contact@mail.arman.is-a.dev",
+    "github": "ItzArman09"
```
Comment (MaskDuck) [inline domains/arman.json:7]:
```suggestion
    "email": "contact@mail.arman.is-a.dev"
```

File: domains/foxie.arman.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Foxie's Homepage by arman",
+  "repo": "https://github.com/ItzArman09/foxie.js",
+  "owner": {
+    "username": "arman",
+    "email": "contact@mail.arman.is-a.dev",
+    "github": "ItzArman09"
```
Comment (MaskDuck) [inline domains/foxie.arman.json:7]:
```suggestion
    "username": "ItzArman09",
    "email": "contact@mail.arman.is-a.dev"
```

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by ItzArman09** [comment]
commited

File: domains/foxie.arman.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Foxie's Homepage by arman",
+  "repo": "https://github.com/ItzArman09/foxie.js",
+  "owner": {
+    "username": "ItzArman09",
+    "email": "contact@mail.arman.is-a.dev"
+  },
+  "record": {
+    "CNAME": "itzarman09.github.io"
```
Comment (MaskDuck) [inline domains/foxie.arman.json:9]: 404?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by ItzArman09** [comment]
1 min

**Comment by ItzArman09** [comment]
check?

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**